setup/setup_vagrant.sh

```diff
@@ -3,12 +3,13 @@
 # It's only tested with ubuntu 14.04
 set -e
 
-REFUGE_PATH=/vagrant/refugerestrooms
+REFUGE_PATH=/vagrant
```

Comment:
This should have been changed alongside 30301d259cbfe16eea1d71ae329a6804ab9150e1.